gateway/src/opcodes/LazyRequest.ts

```diff
@@ -17,8 +15,9 @@ async function getMembers(guild_id: string, range: [number, number]) {
 		throw new Error("range is not a valid array");
 	}
 	// TODO: wait for typeorm to implement ordering for .find queries https://github.com/typeorm/typeorm/issues/2620
+	// TODO: rewrite this, released in 0.3.0
 
-	let members = await getRepository(Member)
+	let members = await (await initDatabase()).getRepository(Member)
```

Comment:
You reinit the database on every lazyrequest?

Reply:
No, initDatabase() returns database instance.